src/components/DiagnosticsBody.vue

```diff
@@ -26,37 +34,62 @@ export default Vue.extend({
           label: 'Category',
         },
       ],
-      rows: [
-        {
-          id:            0,
-          description:   'The ~/.rd/bin directory has not been added to the PATH, so commandline utilities are not configured in your back shell',
-          documentation: 'https://docs.rancherdesktop.io/',
-          category:      'Kubernetes',
-          mute:          false,
-          fixes:         { description: 'You have selected manual PATH configuration, you can let Rancher Desktop automatically configure it.' },
-        },
-        {
-          id:            1,
-          description:   'Are the files under ~/.docker/cli-plugins symlinks to ~/.rd/bin?',
-          documentation: 'https://docs.rancherdesktop.io/',
-          category:      'Kubernetes',
-          mute:          false,
-          fixes:         { description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin sit amet iaculis diam. Nullam ut dolor nec dolor vestibulum viverra id a arcu.' },
-        },
-      ],
     };
   },
+  computed: {
+    numFailed(): number {
+      return this.rows.length;
+    },
+    numMuted(): number {
+      return this.rows.filter(row => (row as DiagnosticsCheck).mute).length;
+    },
+    // npm module timeago-simple is substandard - see https://github.com/mikepenzin/timeago-simple/issues/1
+    // npm module timediff requires as much work as this function uses, so let's stick with it
```

Comment:
Can we put this in a file that is shared, and make `BackendProgress` use it too? Unfortunately, I can't get `Intl.RelativeTimeFormat` to do durations in the way we want.

Reply:
I'm using dayjs() now. It's output is fuzzy, but if the user hovers over the fuzzy time string they get the long-form precise timestamp.